src/api/routes/cron.ts

```diff
@@ -39,6 +42,28 @@ const cron = router({
     });
     await Promise.all(promises);
   }),
+
+  /**
+   * Check the numbers of meetings in progess by each Tencent user id
+   */
+
+  currentMeetingNumbers: procedure.mutation(async () => {
+    const TMuserID: string = "ADMIN"; // change if multiple users used, possibly using hashed TM user Ids
```

Comment:
`TMUserId`. Also, use TM_ADMIN_USER_ID and not a fake id.

Reply:
Do we need to hash the id instead of inputting the actual id?